johnsonandjohnson/Bodiless-JS
# fix (psh): Rerunning init-psh loses SSI on routes.yaml

## Changes
Exclude routes.yaml from copying on running psh-init command.

## How to test
- Add the next line into `examples/test-site/package.json`:
```json
"init-psh": "bodiless-psh-init"
```
- Run `npm run init-psh` from `examples/test-site/`
- Make sure `examples/test-site/.platform/routes.yaml` is created
- Add some changes to routes.yaml
- Run `npm run init-psh` again
- Make sure routes.yaml is not overwritten

## Related Issues
Closes #267
